ui/app/main.go

```diff
@@ -37,15 +37,14 @@ func main() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
-		const elmMake = "elm-make"
-		elmMakeArgs := []string{"src/Main.elm", "--output", "script.js"}
+		elmMakeArgs := []string{"make", "src/Main.elm", "--yes", "--output", "script.js"}
```

Comment:
That's cool. In general we try to bundle relevant code changes together in a PR, so try to do that in the future. That having been said, this `main.go` file is totally unimportant, so no worries on changing it here.